lib/schemas/FieldSchema.js

```diff
@@ -33,6 +35,16 @@ module.exports = makeSchema(
       { key: 'abc', choices: [{ label: 'Red', value: '#f00' }] },
       { key: 'abc', choices: 'mobile' },
       { key: 'abc', type: 'loltype' },
+      { key: 'abc', children: [] },
+      {
+        key: 'abc',
+        children: [{ key: 'def', children: [] }]
+      },
+      {
+        key: 'abc',
+        children: [{ key: 'def', children: [{ key: 'dhi' }] }],
+        skip: true
```

Comment:
The name `skip` looks like part of the field definition, but it's an internal meta field used to bypass the example tests. Can we rename for it to look more like an "internal" thing, such as `_skipTest`?

Reply:
Sure! I also moved it into a constant which will be nice as soon as it comes up a bunch of places.